### PR TITLE
Embedding Audio to Nodes

### PR DESCRIPTION
**Pull request by candideu**
Hello! I made a feature request a few days ago about embedding media - https://github.com/adityamukho/hivemind/issues/20. I decided to revisit the idea and see if I could contribute.

For now, I think I was able to figure out embedding audio from an external URL ([such as this](https://www.kozco.com/tech/piano2-CoolEdit.mp3)). Unfortunately, I wasn't able to test this out my self and build the app on my own. Please let me know if this works!

### CODE REVIEW

File: components/mindmap/EventDetail.js [hunk 1]
```diff
@@ -100,6 +100,13 @@ const EventDetail = ({ event, setNode }) => {
               content.innerHTML = `<b>Content:</b> ${d.content}`
               rows.appendChild(content)
             }
+            
+            if (d.audio) {
+              const content = document.createElement('div')
+              audio.classNames = ['row']
+              audio.innerHTML = `<b>Audio:</b> ${d.content}`
+              rows.appendChild(content)
```
Comment (adityamukho) [inline components/mindmap/EventDetail.js:108]:
```suggestion
              const audio = document.createElement('div')
              audio.classNames = ['row']
              audio.innerHTML = `<b>Audio:</b> ${d.audio}`
              rows.appendChild(audio)
```

**Review by adityamukho — CHANGES_REQUESTED** [review]

**Comment by adityamukho** [comment]
Please change base branch to `development`. Also, have suggested a few changes.

File: components/mindmap/menu-items/view.js [hunk 2]
```diff
@@ -100,6 +100,18 @@ const PopperCard = ({ el, poppers }) => {
...
             </>
           ) : null}
+          {data.audio ? (
+            <>
+              <Row>
+                <h5>Audio</h5>
+              </Row>
+              <Row>
+                <audio controls>
+                <source src="{data.audio}">
```
Comment (adityamukho) [inline components/mindmap/menu-items/view.js:110]:
```suggestion
                <source src="{data.audio}"/>
```

Comment (adityamukho) [reply components/mindmap/menu-items/view.js:110]: Required to fix failed deployment: https://github.com/adityamukho/hivemind/pull/21#issuecomment-987766811

**Comment by candideu** [comment]
Okay, I've implemented the changes and rebased. See this new PR: https://github.com/adityamukho/hivemind/pull/22

File: components/mindmap/menu-items/edit.js [hunk 3]
```diff
@@ -170,6 +172,18 @@ const PopperCard = ({ el, poppers }) => {
...
               />
             </FormGroup>
+            <FormGroup>
+              <Label for="audio">Audio (URL)</Label>
+              <Input
+                type="url"
+                name="audio"
+                id="audio"
+                value={audio}
+                maxLength="2,048"
```
Comment (adityamukho) [inline components/mindmap/menu-items/edit.js:182]:
```suggestion
                maxLength="2048"
```

The numeric value with a comma gets truncated to 2 when parsed.

**Comment by adityamukho** [comment]
Follow-up at #22

**Pull request closed without merging**